mula/scheduler/models/plugin.py

```diff
@@ -19,3 +19,4 @@ class Plugin(BaseModel):
     produces: list[str]
     cron: str | None = None
     interval: int | None = None
+    runon: list[str] | None = None
```

Comment:
I guess we could support both the string and the multiple option, however when we create a schema for the boefjes Im guessing it would be best to go for just the list.